<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="camera">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>671</width>
     <height>301</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="picbtn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>380</y>
     <width>231</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgba(255, 241, 36, 0.8);
	border-radius: 20px;
	border-color: rgb(155, 142, 149, 0.8);	
	border-style : solid;
	border-width : 0.5px;
}</string>
   </property>
   <property name="text">
    <string>촬영하기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="picbtn_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>380</y>
     <width>231</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	background-color: rgba(222,222, 222, 0.8);
	border-radius: 20px;
	border-color: rgb(155, 142, 149, 0.8);	
	border-style : solid;
	border-width : 0.5px;
}</string>
   </property>
   <property name="text">
    <string>뒤로가기</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
